### ISSUE TITLE
Doesn't work

### ISSUE THREAD

**Issue by zjw102** [liuyyy111/Point-RAE#8]
Hello, what changes have you made, and why I don't have any response when I run the code. It's stuck all the time, and other code works normally, not the server's problem

**Comment by tianbao-anw** [comment]
Hello, I'm not quite sure what you mean by "don't have any response." Are you referring to not getting any logs or not receiving a specific type of output? I haven’t encountered this situation before, but if you can provide more details, I’ll do my best to help you troubleshoot it.

**Comment by liuyyy111** [comment]
Hello, I'm not quite sure what you mean by "don't have any response." Are you referring to not getting any logs or not receiving a specific type of output? I haven’t encountered this situation before, but if you can provide more details, I’ll do my best to help you troubleshoot it.

**Comment by zjw102** [comment]
Thank you very much for your reply, I found the problem and the problem is with lightly. I've found that when using pip install -r requirements.txt, lightly causes the latest torch to be downloaded, which causes the code to run without a response. When I settled it out lightly, everything worked fine. Thank you again for your reply.

**Comment by zjw102** [comment]
When I run Stage II, pretrain Point-RAE on ShapeNet, I get this RuntimeError: Point_RAE: Error(s) in loading state_dict for ACTPromptedDiscreteVAEwithVIT:
        Missing key(s) in state_dict: "visual_prompt_token", "visual_prompt_pos", "deep_prompt_tokens", "deep_prompt_pos". 
Excuse me, is that the case? When I looked at the build_visual_embedding in the dvae.py, I saw that in Stage I, the num_prompt_token was 0 and the use_deep_prompt was False. So is it because of this part?
